hw/mcu/dialog/da1469x/src/hal_flash.c

```diff
@@ -382,5 +382,11 @@ da1469x_hff_sector_info(const struct hal_flash *dev, int idx,
 static int
 da1469x_hff_init(const struct hal_flash *dev)
 {
+#if defined (MYNEWT_VAL_MCU_QSPIC_BURSTCMDA_INIT_VAL)
```

Comment:
Maybe move these settings to always execute from RAM? I.e. within another function with section attribute macro 'sec_text_ram_core' defined.

Reply:
@mkiiskila Took your suggestion. Also, added the ability to configure MCU_QSPIC_CTRLMODE_REG and re-added disabling interrupts. 

After more testing I'm confident this fix is safe.